src/components/ContactUsForm/index.jsx

```diff
@@ -0,0 +1,127 @@
+import React, { useState } from "react";
+import { useTranslation } from "react-i18next";
+import { Col, Row } from "antd";
+import { RiSendPlaneFill } from "react-icons/ri";
+import "./index.css";
+
+const ContactUsForm = () => {
+  const [t] = useTranslation();
+  const [formValue, setFormValue] = useState({
+    name: "",
+    email: "",
+    message: "",
+  });
+
+  // The change in isSubmitted state triggers submission message
+  const [isSubmitted, setIsSubmitted] = useState(false);
+
+  // Converts object to proper form data(string)
+  const encode = (data) => {
+    return Object.keys(data)
+      .map(
+        (key) => encodeURIComponent(key) + "=" + encodeURIComponent(data[key])
+      )
+      .join("&");
+  };
+
+  const handleChange = (e) => setFormValue({ [e.target.name]: e.target.value });
+
+  // application/x-www-form-urlencoded stands for POST data at HTTP.
+  // body represents HTTP message and send data to server in a query string format.
+  // encode is used for passing the string to the body
+  // setTimeOut stands for showing submission message about a time after submit
+  const handleSubmit = (e) => {
+    fetch("/", {
+      method: "POST",
+      "data-netlify": "true",
+      headers: { "Content-Type": "application/x-www-form-urlencoded" },
+      body: encode({ "form-name": "contact", ...formValue }),
+    }).then(() => {
+      setTimeout(() => setIsSubmitted(false), 5000);
+      setIsSubmitted(true);
+      setFormValue({
+        name: "",
+        email: "",
+        message: "",
+      }).catch((error) => console.log(error));
+
+      e.preventDefault();
+    });
+  };
+
+  return (
+    <div className="contactUsFormContainer">
+      <form onSubmit={handleSubmit}>
+        <Col>
+          <div
+            className="contactUsForm__submissionMessage"
+            style={
+              isSubmitted ? { visibility: "visible" } : { visibility: "hidden" }
```

Comment:
style={{visibility: isSubmitted ? "visible" : ... }}

Reply:
Thanks for your review, it's format changed and become cleaner, PTAL.